packages/components/src/components/carousel/carousel.scss

```
@@ -79,10 +79,17 @@
     }
     &__icon {
         cursor: pointer;
+        background: #f2f3f4;
```

Generally, we don't use hardcoded color values unless we have to. please use CSS variables for colors:
```suggestion
        background: var(--icon-grey-background);
```